scripts/kube-init.sh

```diff
@@ -44,10 +44,19 @@ else
 fi
 docker --version
 
-# Get the latest stable version of kubernetes
+# Get the latest stable version of kubernetes, this is not always what minikube
+# installs per default
+# See: https://github.com/kubernetes/minikube/master/pkg/minikube/constants/constants.go
```

Comment:
nit:

https://github.com/kubernetes/minikube/blob/master/pkg/minikube/constants/constants.go

Reply:
Thanks, I update the link.